src/GUI/Overlay/OverlayEventHandler.cpp

```diff
@@ -109,3 +111,10 @@ bool OverlayEventHandler::textInputEvent(const PlatformTextInputEvent *event) {
     nk_input_glyph(_context, glyph);
     return nk_item_is_any_active(_context);
 }
+
+bool OverlayEventHandler::nativeEvent(const PlatformNativeEvent *event) {
+    // Here we're assuming the native event is coming from SDL
+    const SDL_Event *sdlEvent = reinterpret_cast<const SDL_Event *>(event->nativeEvent);
```

Comment:
Doing `static_cast` is perfectly legal X).

Reply:
Not sure why I was convinced I had to do when casting from void* to anything else X)